std/experimental/allocator/package.d

```diff
@@ -652,12 +720,12 @@ T[] makeArray(T, Allocator)(auto ref Allocator alloc, size_t length,
         }
         for (; i < length; ++i)
         {
-            emplace!T(result.ptr + i, init);
+            emplaceRef(() @trusted { return *(result.ptr + i); } (), init);
         }
     }
     else
     {
-        fillWithMemcpy(result, init);
+        () @trusted { fillWithMemcpy(result, init); } ();
```

Comment:
memcpy can corrupt user memory and/or lead to memory leaks, if used incorrectly. Every usage must be verified manually.